src/clj/orcpub/email.clj

```diff
@@ -27,7 +27,7 @@
 (defn email-cfg []
   {:user (environ/env :email-access-key)
    :pass (environ/env :email-secret-key)
-   :host "email-smtp.us-west-2.amazonaws.com"
+   :host (environ/env :email-server-url)
    :port 587})
```

Comment:
might as well add the port as a (environ/env) too.

Reply:
Yeah, why not. Exposing